ui/src/app/richskill/service/rich-skill.service.spec.ts

```diff
@@ -306,6 +306,23 @@ describe("RichSkillService", () => {
     })
   })
 
+  it("libraryExport should return", () => {
+    RouterData.commands = []
+
+    // Act
+    const result$ = testService.libraryExport()
+
+    // Assert
+    result$.subscribe( (data: string) => {
+      expect(RouterData.commands).toEqual([]) // No Errors
```

Comment:
It doesn't look like RouterData.commands ever gets assigned a new value.  Also, the subscription data parameter is never used.  Perhaps that's what was intended.

Reply:
I don't recall right now, but since there is an expect() statement in the callback, you might run into occasional test failures.  If that happens, we may have to mark the test case as asynchronous.  There are >=2 ways to deal with that situation.

Reply:
I have added the fakeAsync() so it doesn't fail.